.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Run tests
         run: yarn test
       - name: Run E2E tests
-        run: yarn test:e2e:docker
+        run: yarn test:e2e
```

Comment:
- Outdated rest-api.md
    - Invalid interface for ELBlockSnapshot
    - lastChangedBlockHash - not represented in API
    - v1/modules - Module inrterface not full
    - /v1/modules/{module_id}/validators/validator-exits-to-prepare/ if not exist NO should be 404 exception